.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: tests
+on: { pull_request: {} }
+
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    strategy:
```

see below comment first (on time dilation), but we can add the dilation factor here:

```suggestion
    strategy:
    env:
      TEST_TIME_DILATION_FACTOR: 10 # or some other value
```